src/cmd.zig

```diff
@@ -181,7 +181,7 @@ const Cmd = struct {
     metrics_port: u16,
     log_file: ?[]const u8,
     tee_logs: bool,
-    subcmd: union(enum) {
+    subcmd: ?union(enum) {
```

Comment:
Previously it did not need to be optional? I think it would be cleaner to leave it the old way.

Reply:
It might be cleaner, but it is more complicated and less flexible, adding multiple ways doing the same thing, such that the core features are less discoverable.